index.Rmd

```diff
@@ -8,6 +8,7 @@ output:
     theme: spacelab
     toc: false
     toc_depth: 4
+    fig_caption: false
```

Comment:
@carriewright11 I added this because the alt text was showing up under the zoom screenshots and looked like it might have been unintentional - please ignore if that's not the case!

Reply:
yeah, that looks better- thanks!